doc/howtos/bluetoothspeaker.rst

```diff
@@ -0,0 +1,141 @@
+Bluetooth Speaker
+#################
+
+How to setup your intel-iot-refkit image to function as a bluetooth speaker
```

Comment:
Well Joule doesn't have a speaker (or even sound card) so you have to connect something behind usb or hdmi. if you have speakers in your monitor and connect through HDMI you 're fine. This is a stand alone fully functional thing. In my setup on my table you pair for example your phone and and its ready. If I play from my spotify app the music is streaming and heard.